pallets/automation-time/src/lib.rs

```diff
@@ -460,6 +567,25 @@ pub mod pallet {
 			10_000
 		}
 
+		fn transfer(
+			sender: T::AccountId,
+			recipient: T::AccountId,
+			amount: <<T as pallet::Config>::Currency as frame_support::traits::Currency<<T as frame_system::Config>::AccountId>>::Balance,
+		) -> Weight {
+			
+			// TODO: jzhou fix later
+
+			// <<T as pallet::Config>::Currency as frame_support::traits::Currency<<T as frame_system::Config>::AccountId>>::Balance;
+			// <T as pallet::Config>::Balance;
+			<T as Config>::Currency::transfer(&sender, &recipient, amount, ExistenceRequirement::KeepAlive);
+
+			// let message = Vec::new();
+			// message.push("Sent {amount} tokens from {from} to {to}");
+			// Self::deposit_event(Event::Notify{ message });
```

Comment:
is there an easy way to construct a message if i want to emit an event to say the transfer is complete? or do i need to take each letter and convert into a u8 number and insert into vector?

also, is it fine if i just emit event saying `Transfer complete for : {providedId}`? or does anyone else need more info?